README.rst

```diff
@@ -141,10 +143,13 @@ One may also use other element types::
   a = DisjointSets{AbstractString}(["a", "b", "c", "d"])
   union!(a, "a", "b")
   in_same_set(a, "c", "d")
+  root_union!(a, "a", "d")
   push!(a, "f")
 
 
-Note that the internal implementation of ``IntDisjointSets`` is based on vectors, and is very efficient. ``DisjointSets{T}`` is a wrapper of ``IntDisjointSets``, which uses a dictionary to map input elements to an internal index.
+Note that the internal implementation of ``IntDisjointSets`` is based on vectors, and is very efficient. ``DisjointSets{T}`` is a wrapper of ``IntDisjointSets``, which uses a dictionary to map input elements to an internal index. 
+As a result, ``union!(a, "a", "b")`` and ``root_union!(a, "a", "d")`` returns the indices of the
+root elements rather than the root elements themselves.
```

Comment:
I'm wondering...

1. Is there a way to go from index to element?
2. If someone is manipulating non-integer `DisjointSets`, is returning the resulting index is useful?  Can they use that index in further processing?  How?